llvm/lib/Target/WebAssembly/WebAssemblyISelLowering.cpp

```diff
@@ -1211,6 +1211,15 @@ WebAssemblyTargetLowering::LowerCall(CallLoweringInfo &CLI,
         }
       }
     }
+
+    // If outgoing arguments are passed via the stack, we cannot tail call
+    for (const ISD::OutputArg &Out : CLI.Outs) {
```

Comment:
This is a completely unrelated change. Can it be split out into a separate PR (preferably with a test?)

Reply:
Aha, i just got one item deeper into my queue and discovered #124443, thanks :)
Please just delete this change from the PR.

Reply:
Whoops! 😅